### DISCUSSION TITLE
Is this project active?

### DISCUSSION THREAD

**Discussion by ghost** [AvaloniaUI/Avalonia#9812] [Q&A] [answered]
Hi all.

I want to help, but I want to know if this project is active. *Is this project active?*

**Comment by maxijabase** [comment, marked as answer]
Yes, Avalonia's development is pretty active for the time being.

**Reply by ghost** [reply]
Hi maxijabase.

> Yes, Avalonia's development is pretty active for the time being.

thank you for feedback.
